### DISCUSSION TITLE
Technical reviewer needed

### DISCUSSION THREAD

**Discussion by aitoehigie** [fastapi/fastapi#6905] [Questions] [answered]
Hello,
I dont know if this is the right place to ask this but is there anyone here up to reviewing a technical resource on FastAPI?

Thanks

**Comment by Mause** [comment, marked as answer]
There's a jobs channel in the discord server, GitHub issues aren't the best place for job offers

**Comment by aitoehigie** [comment]
Thank you.

**Comment by tiangolo** [comment]
Thanks for the help here @Mause ! 👏  🙇 

Thanks for coming back and closing the issue @aitoehigie  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
